.codecov.yml

```diff
@@ -0,0 +1,21 @@
+codecov:
+  notify:
+    require_ci_to_pass: yes
+  strict_yaml_branch: main  # only use the latest copy on main branch
+
+coverage:
+  precision: 2
+  round: down
+  range: "80...100"
+  status:
+    project:
+      default:
+        enabled: yes
+        target: 90%
+    patch:
+      default:
+        enabled: yes
+        target: 95%
+
+ignore:
+  - "tests/**/*"
```

Comment:
Was this required?  testutils should have 90+% coverage and I don't believe there should be statements to cover in integration tests.